sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -54,6 +41,42 @@ export class BatchingReceiver extends MessageReceiver {
    */
   constructor(context: ClientEntityContext, options?: ReceiveOptions) {
     super(context, ReceiverType.batching, options);
+
+    this._batchingReceiverLite = new BatchingReceiverLite(
+      context,
+      async (abortSignal?: AbortSignalLike): Promise<MinimalReceiver> => {
+        let lastError: Error | AmqpError | undefined;
+
+        const rcvrOptions = this._createReceiverOptions(false, {
+          onError: (context) => {
+            lastError = context.error || context?.session?.error || context?.receiver?.error;
+          },
+          onSessionError: (context) => {
+            lastError = context.error || context?.session?.error || context?.receiver?.error;
```

Comment:
What is the reason for the fall back? Historically, we have only checked for errors in `context.session` in case of the session_error event and `context.receiver` for the `receiver_error` event.

Reply:
Will fix.

Reply:
Trimmed it down.